tests/fixtures/debian/kafka-streams-examples/standalone-config.yml

```diff
@@ -0,0 +1,51 @@
+---
+version: '2'
+services:
+  zookeeper:
+    image: confluentinc/cp-zookeeper:3.2.0
```

Comment:
`latest` is fine for tests as the local build you are testing against will always have `latest` pointing to the current version.